janus_server/src/binary_utils.rs

```diff
@@ -60,3 +70,149 @@ pub fn datastore<C: Clock>(
     }
     Ok(Datastore::new(pool, Crypter::new(ds_keys), clock))
 }
+
+/// Options for Janus binaries.
+pub trait BinaryOptions: StructOpt + Debug {
+    /// Returns the common options.
+    fn common_options(&self) -> &CommonBinaryOptions;
+}
+
+// Common options that are used by all Janus binaries.
```

Comment:
This is deliberately not a doccomment to work around [a known issue in structopt](https://github.com/TeXitoi/structopt/issues/333). It's not clear if/when that will be fixed, but in any case, I learned today that [structopt has been deprecated in favor of functionality built into `clap`](https://github.com/TeXitoi/structopt/issues/525).

Reply:
How about this?
```suggestion
#[cfg_attr(doc, doc = "Common options that are used by all Janus binaries.")]
```

Reply:
That works, thanks!